shell/platform/darwin/macos/framework/Source/FLEExternalTextureGL.mm

```diff
@@ -0,0 +1,85 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FLEExternalTextureGL.h"
+
+#import <AppKit/AppKit.h>
+#import <CoreVideo/CoreVideo.h>
+#import <OpenGL/gl.h>
+
+@implementation FLEExternalTextureGL {
+  CVOpenGLTextureCacheRef _textureCache;
+  CVPixelBufferRef _pixelBuffer;
+  id<FLETexture> _texture;
+}
+
+- (instancetype)initWithFLETexture:(id<FLETexture>)texture {
+  self = [super init];
+  if (self) {
+    _texture = texture;
+    _pixelBuffer = nil;
+    _textureCache = nil;
+  }
+  return self;
+}
+
+static void OnGLTextureRelease(CVPixelBufferRef pixelBuffer) {
```

Comment:
Nit: please put this up above the @implementation, rather than embedded into the class definition.